content/features/04-graph-traffic-visualization.adoc

```diff
@@ -3,8 +3,10 @@ title: "Graph: Traffic Animation"
 date: 2018-06-20T19:04:38+02:00
 draft: false
 image: features/graph-traffic-animation.gif
-subTitle: "Animating your traffic flow"
+subTitle: "Visualizing your traffic flow"
 weight: 04
 ---
 
-The traffic animation provides a visualization of traffic between your applications as represented as circles flowing between them. More circles means more **requests**, faster moving circles means a faster **response time**, and red triangles flowing between the applications represents **errors**. 
+For HTTP traffic, circles represent successful requests while red diamonds represent errors. More shapes means more **requests** and faster moving shapes means a faster **response time**.
+
+TCP traffic is represented by offset circles where faster moving circles means a faster traffic speed.
```

Comment:
circles/circle (it's followed by "means" which is not plural)

Reply:
I think I might be missing something. What is the problem with having circles be plural?

Reply:
if the word "circles" is plural, then the following "means" need to be in plural "mean" too, no?
"circles mean", or alternatively "circle means"
Or there is some other English thing here that I'm missing :)

Reply:
@JStickler Can you help us out with this?